returnn/torch/updater.py

```diff
@@ -80,7 +81,7 @@ class Updater(object):
     Wraps a torch.optim.Optimizer, and extends it by some further functionality.
     """
 
-    def __init__(self, config, network, initial_learning_rate=1.0):
+    def __init__(self, config, network, device, initial_learning_rate=1.0):
```

Comment:
I think we should enforce kwargs-only usage here.
```suggestion
    def __init__(self, *, config, network, device, initial_learning_rate=1.0):
```